### PR TITLE
391 Rems external id

### PR DESCRIPTION
**Pull request by aleuk0**

### CODE REVIEW

File: core/models/access.py [hunk 1]
```diff
@@ -185,6 +185,15 @@ def clean(self):
         help_text="Was the entry generated automatically, e.g. by REMS?",
     )
 
+    application_id = models.IntegerField(null=True, help_text="REMS application ID.")
```
Comment (moustaphacheikh) [inline core/models/access.py:188]: Are these mandatory?  blank =False will force the user to submit them when they create a new access instance

Comment (aleuk0) [reply core/models/access.py]: done

**Comment by moustaphacheikh** [comment]
@aleuk0 It worked as expected. i tested the end-to-end process. great work

**Review by moustaphacheikh — APPROVED** [review]
@aleuk0 It worked as expected. i tested the end-to-end process. great work

**Pull request merged**